modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-mixing-length-physics.i

```diff
@@ -0,0 +1,89 @@
+Re = 1e4
+von_karman_const = 0.2
+
+D = 1
+rho = 1
+bulk_u = 1
+mu = '${fparse rho * bulk_u * D / Re}'
+
+advected_interp_method = 'upwind'
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 5
+    ymin = 0
+    ymax = '${fparse 0.5 * D}'
+    nx = 20
+    ny = 10
+    bias_y = '${fparse 1 / 1.2}'
+  []
+[]
+
+[Physics]
+  [NavierStokes]
+    [WCNSFVFlow]
+      [flow]
+        compressibility = 'incompressible'
+
+        density = ${rho}
+        dynamic_viscosity = ${mu}
+
+        initial_velocity = '1 1 0'
+        initial_pressure = 0.0
+
+        inlet_boundaries = 'left'
+        momentum_inlet_types = 'fixed-velocity'
+        momentum_inlet_function = '1 0'
+
+        wall_boundaries = 'top bottom'
+        momentum_wall_types = 'noslip symmetry'
+
+        outlet_boundaries = 'right'
+        momentum_outlet_types = 'fixed-pressure'
+        pressure_function = '0'
+
+        momentum_advection_interpolation = ${advected_interp_method}
+        mass_advection_interpolation = ${advected_interp_method}
+      []
+    []
```

Comment:
Hey @tanoret , what do you think about this syntax? Can you envision your current two-phase/turbulence routines in it? I think it is pretty flexible.

Reply:
see the whole input, this only shows flow

Reply:
Normally for the two-phase flow we will need to add mixture materials as materials, which is doable by extending the current actions, but it would be much harder to set up the kernels fo each case required for two-phase flow. Also setting up solidification and melting will be quite hard in this configuration...

Reply:
If the equations for flow are close enough, then I can see if we can make do with using the same Physics object

likely not though. I think we will want to have a dedicated two phase block "[WCNSFV2PhaseFlow]" for example